client/src/components/forms/submit-button.component.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { describe, expect, it } from 'vitest';
+import { SubmitButton } from './submit-button.component';
+import { render, screen } from '~/test/utils';
+import { GBIcon } from '../../i18n/icons';
+describe('Submit Button', () => {
+  it('should render a submit button with text only', () => {
+    render(<SubmitButton isDisabled>Test text</SubmitButton>);
+    expect(screen.getByRole('button', { name: 'Test text' })).toBeInTheDocument();
+  });
```

Comment:
You've set the button to be disabled but don't test that it is, so I suggest you remove the `isDisabled` attribute.